docs/mctpd.md

```diff
@@ -89,5 +89,13 @@ busctl call xyz.openbmc_project.MCTP  /xyz/openbmc_project/mctp/1/11 \
 
 Removes the MCTP endpoint from `mctpd`, and deletes routes and neighbour entries.
 
+## D-Bus /xyz/openbmc_project/mctp/links/<InterfaceName>
```

Comment:
Some bikeshedding that may come out of #40: `links` or `interfaces` (or something else?)

Reply:
I updated code to follow the propose #40.